src/data-loader.ts

```
@@ -522,7 +523,7 @@ export async function getEarliestTimestamp(filePath: string): Promise<Date | nul
 	catch (error) {
 		// Log file access errors for diagnostics, but continue processing
 		// This ensures files without timestamps or with access issues are sorted to the end
-		logger.debug(`Failed to get earliest timestamp for ${filePath}:`, error);
+		logger.debug(i18n.tp('warnings.failedToGetTimestamp', { file: filePath }), error);
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)